### PR TITLE
create Stompy set-up

### PR DESCRIPTION
**Pull request by budzianowski**

### CODE REVIEW

**Comment by budzianowski** [comment]
We should add the info as well here:
https://github.com/isaac-sim/IsaacLab/pull/453/files#diff-52508297e69cd93e383189d2a76fb34343f718b92fcea685f2a6f7511bf8582f
and here:
https://github.com/isaac-sim/IsaacLab/pull/453/files#diff-1bc4763f7f7050d5e8f165c8474b16095d813f72b217c4ff51f92b227c294e15

File: source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+"""Configuration for Stompy."""
+
+import omni.isaac.lab.sim as sim_utils
+from omni.isaac.lab.actuators import ImplicitActuatorCfg
+from omni.isaac.lab.assets.articulation import ArticulationCfg
+
+STOMPY_CFG = ArticulationCfg(
+    # Spawn stompy from URDF
+    spawn=sim_utils.UsdFileCfg(
+        usd_path="/home/dpsh/isaacs_sim/sim/sim/stompy/robot_fixed/robot_fixed.usd",
```
Comment (budzianowski) [inline source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy.py:15]: shouldn't we use `source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/stompy/stompy/robot_fixed/robot_fixed.usd`?

Comment (wuallenwu) [reply source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy.py:15]: you're right i forgot to git add that file in the cleanup

**Review by wuallenwu — APPROVED** [review]

File: source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            solver_position_iteration_count=8,
+            solver_velocity_iteration_count=4,
+        ),
+    ),
+    init_state=ArticulationCfg.InitialStateCfg(
+        pos=(0.0, 0.0, 1.25),
+        joint_pos={
+            "torso_roll": -0.502,
+            # left arm
+            "left_shoulder_pitch": -0.251,  # Corrected from -0.502
```
Comment (budzianowski) [inline source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy.py:37]: What's the meaning of the comments?

File: source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+import omni.isaac.lab.sim as sim_utils
+from omni.isaac.lab.actuators import ImplicitActuatorCfg
+from omni.isaac.lab.assets.articulation import ArticulationCfg
+
+STOMPY_CFG = ArticulationCfg(
+    # Spawn stompy from URDF
+    spawn=sim_utils.UsdFileCfg(
+        usd_path="/home/dpsh/isaacs_sim/sim/sim/stompy/robot_fixed/robot_fixed.usd",
+        activate_contact_sensors=True,
```
Comment (budzianowski) [inline source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy.py:16]: Where do these parameters come from?

File: source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/stompy/rough_env_cfg.py [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+        # Randomization
+        # TODO: LOOK FOR HYPERPARAMETERS
+        self.events.push_robot = None
+        self.events.add_base_mass = None
+        self.events.reset_robot_joints.params["position_range"] = (1.0, 1.0)
+        self.events.base_external_force_torque.params["asset_cfg"].body_names = [
+            ".*link_upper_limb_assembly_7_dof_1_torso_1_bottom_skeleton_1"
+        ]
+        self.events.reset_base.params = {
```
Comment (budzianowski) [inline source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/stompy/rough_env_cfg.py:129]: Are these default values?

**Review by budzianowski — COMMENTED** [review]
Looks good! Let's push this in and start larger training. Can you write a readme here with a simple instructions what are the steps to add the model?

File: source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/stompy/rough_env_cfg.py (hunk 4 above)
Comment (wuallenwu) [reply source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/stompy/rough_env_cfg.py:129]: yep they were using a torso link so i matched it as best i could

File: source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy.py (hunk 2 above)
Comment (wuallenwu) [reply source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy.py]: fixed

File: source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy.py (hunk 3 above)
Comment (wuallenwu) [reply source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy.py:16]: fixed

**Comment by wuallenwu** [comment]
> Looks good! Let's push this in and start larger training. Can you write a readme here with a simple instructions what are the steps to add the model?

i updated the initial readme that has the instructions for trianing and playing

**Pull request merged**